IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedBldgElements/Entities/IfcArchElement/Documentation.md

```diff
@@ -0,0 +1 @@
+A unitary curved structure.
```

Comment:
I like Meriam Webster: "a typically curved structural member spanning an opening and serving as a support"

https://www.merriam-webster.com/dictionary/arch

Reply:
I agree but I just used the definition from IfcElementAssembly/ARCH and added "unitary". Should we change there as well then?

Reply:
> I like Meriam Webster: "a typically curved structural member spanning an opening and serving as a support"
> 
> https://www.merriam-webster.com/dictionary/arch

Please check now.